src/utils/renderer.js

```diff
@@ -0,0 +1,30 @@
+/** @jsx jsx */
+import { Link as ExternalLink, jsx } from 'theme-ui'
+import { createElement } from 'react'
+import rehypeReact from 'rehype-react'
+
+import { Link as AnchorLink } from 'react-scroll'
+import InternalLink from '../components/link'
+
+const extRegex = /^(https?:)?\?\?/
+
+const MdLink = ({ href, children, external, ...props }) =>
+  href[0] === '#' ? (
+    <AnchorLink href={href} to={href.substring(1)} {...props} smooth={true} duration={400} hashSpy={true}>{children}</AnchorLink>
+  ) : extRegex.test(href) || external ? (
+    <ExternalLink href={href} target='_blank' rel='nofollow noopener noreferrer' {...props}>{children}</ExternalLink>
+  ) : (
+    <InternalLink to={href} {...props}>{children}</InternalLink>
+  )
+
+function render(options) {
+  rehypeReact.call(this, {
+    createElement,
+    components: {
+      a: MdLink,
```

Comment:
Maybe we can use primary/accent for h1/h2 in the markdown renderer (use theme-ui header component)? All the titles and headers are white, so it's currently hard to distinguish between different problems at a glance

Reply:
since the publishing date is already in primary (i think), we may even want to consider a color like our earlier palette's green as a md problem title, not sure